src/QueueReceiver.Infrastructure/ServiceCollectionSetup.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Azure.ServiceBus;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using QueueReceiver.Core.Interfaces;
+using QueueReceiver.Core.Services;
+using QueueReceiver.Infrastructure.Data;
+using QueueReceiver.Infrastructure.Repositories;
+
+namespace QueueReceiver.Infrastructure
+{
+    public static class ServiceCollectionSetup
+    {
+        public static readonly LoggerFactory _myLoggerFactory =
```

Comment:
Naming and formatting.

        public static readonly LoggerFactory MyLoggerFactory =
            new LoggerFactory(new []
            {
                new Microsoft.Extensions.Logging.Debug.DebugLoggerProvider()
            });